src/transport/grpc/convert.rs

```diff
@@ -62,6 +64,68 @@ pub enum MessageHubProtoError {
     DateConversion(String),
 }
 
+/// Map a received message hub property to an optional astarte type
+pub fn map_property_to_astarte_type(
```

Comment:
Is this function need to be `pub`? Maybe is keep only visible to the crate
```suggestion
pub(crate) fn map_property_to_astarte_type(
```

Reply:
Yes they can probably be carte public if the message hub does not use them, and I don't think it does.